packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/translations/Messages_en_GB.json

```diff
@@ -21,7 +21,6 @@
   "pci_projects_project_serving_namespace_models_storage_path": "Object Storage Repository",
   "pci_projects_project_serving_namespace_models_storage_scale_min": "Minimum instances",
   "pci_projects_project_serving_namespace_models_storage_scale_max": "Maximum instances",
-  "pci_projects_project_serving_namespace_models_add_autoscaling_info": "During the beta-testing phase, auto-scaling is not customisable.",
```

Comment:
Removed in https://github.com/ovh/manager/commit/d9f174d04de5603f8f37f79ae23cec24ec4cdfb7#diff-67ead60f2ffcb8c1af81da722a4be61eL25